desafio-03/phewill/palindromos.php

```diff
@@ -0,0 +1,20 @@
+<?php
```

Comment:
Coloque os seus dados como comentários. Se puder, prepare o arquivo pra ser executável via php command line. Uma boa idéia é colocar também o link pro repl.it (como voce colocou no README.md).